docs/Troubleshooting.md

```diff
@@ -138,3 +138,99 @@ and is a JSON object generated by the Quilt build system.
 
     `$ aws cloudformation describe-stack-events --stack-name QUILT_STACK_NAME`
 
+## Collecting logs
```

Comment:
```suggestion
## Collect logs to report a bug
```

Note that the three steps from the section just above this one are all relevant logs and we should consolidate those steps into this section.

All of these steps are of course laborious for customers but let's get this PR over the line first and then as and when possible move to a single script like `get-quilt-logs --stack-name MY_QUILT_STACK` in GitHub or better.